ghost/core/core/server/adapters/cache/TTLMemoryCache.js

```diff
@@ -0,0 +1,50 @@
+const TTLCache = require('@isaacs/ttlcache');
```

Comment:
I'm working on moving this cache to a separate package as it doesn't rly belong in core.